services/director-v2/src/simcore_service_director_v2/modules/db/repositories/services_environments.py

```diff
@@ -34,8 +35,7 @@ async def get_vendor_secrets(
                 vendor_service_version=service_version,
                 normalize_names=True,
             )
-
-            # NOTE: normalize_names = True
+            # NOTE: normalize_names = True  # noqa: ERA001
```

Comment:
can you explain this NOTE? what does this say of anything?

Reply:
I'm not the creator of the note and I don't understand it. Not sure how to deal with it.

Reply:
finally got it. redundant. it's already pretty clear in the function above.